stack/rush-stack-compiler-2.4/includes/tslint.json

```diff
@@ -59,6 +59,7 @@
     "no-duplicate-variable": true,
     "no-empty": true,
     "no-eval": true,
+    "no-floating-promises": true,
```

Comment:
Should we move the `tslint.json` to `rush-stack-compiler-shared` folder?

Reply:
IIRC @iclanton didn't do that because the TSLint rule sets can be different between compiler releases.

---
In reply to: [238999784](https://github.com/Microsoft/web-build-tools/pull/983#discussion_r238999784) [](ancestors = 238999784)

Reply:
Yeah that's why I wanted these separated.